cluster/certificates.go

```diff
@@ -19,86 +19,10 @@ import (
 	"k8s.io/client-go/util/cert"
 )
 
-func SetUpAuthentication(ctx context.Context, kubeCluster, currentCluster *Cluster) error {
+func SetUpAuthentication(ctx context.Context, kubeCluster, currentCluster *Cluster, fullState *RKEFullState) error {
```

Comment:
can we drop RKE from the name? and call the parameter `clusterState`?

Reply:
fixed